src/OrchardCore/OrchardCore.DisplayManagement/Extensions/TagHelperExtensions.cs

```diff
@@ -3,7 +3,7 @@
 
 namespace Microsoft.Extensions.DependencyInjection
 {
-    public static class ServiceCollectionExtensions
+    public static class TagHelperExtensions
```

Comment:
We could have renamed the file instead. Or as aspnetcore does something like `TagHelperServiceCollectionExtensions`. But sometimes it feels to me too long so that's okay.

Anyway here there is already a mix between extensions helpers and what is related to the extensions manager, for example the `ThemeExtensionInfo.cs` file in the same folder.

Reply:
yes. not sure why I changed it. Any way, I renamed it to `ServiceCollectionExtensions`.